leasing/migrations/0084_rent_old_dwellings_in_housing_companies_price_index_type.py

```diff
@@ -17,7 +17,7 @@ class Migration(migrations.Migration):
             name="old_dwellings_in_housing_companies_price_index_type",
             field=enumfields.fields.EnumField(
                 blank=True,
-                enum=leasing.enums.OldDwellingsInHousingCompaniesPriceIndexType,
+                enum=leasing.enums.PeriodicRentAdjustmentType,  # This has been renamed later
```

Comment:
I edited this previous migration manually, which is a dirty trick. But I don't remember how the imports were handled properly in migrations to handle issues like these.